observer/storage/blockmap.go

```diff
@@ -5,7 +5,7 @@ import (
 )
 
 type BlockMap struct {
-	heights map[int]*Block
+	heights map[interface{}]*Block
 	lock    sync.RWMutex
```

Comment:
what's the purpose to use interface instead of int?

Reply:
Because bitcoin id is 0, if I keep the coin with int in the database, I cannot use this field as a primary key because zero is null. So, we pass the int for the database model, but in the database going to store as a string. The GORM does the casting.

https://github.com/trustwallet/blockatlas/pull/451/files#diff-517d2699fdea1888a8a32a0462a21152R6